hack/verify-lifted.sh

```diff
@@ -0,0 +1,20 @@
+#!/usr/bin/env bash
+
+set -o errexit
+set -o nounset
+set -o pipefail
+
+SCRIPT_ROOT=$(dirname "${BASH_SOURCE[0]}")/..
+
+doc_file="pkg/util/lifted/doc.go"
+"$SCRIPT_ROOT/hack/update-lifted.sh"
```

Comment:
Is it possible to create a temporary file like any other verify script and then restore it after the comparison?

Reply:
ok~